cni/netconfig.go

```diff
@@ -67,6 +67,11 @@ type NetworkConfig struct {
 		Address       string `json:"ipAddress,omitempty"`
 		QueryInterval string `json:"queryInterval,omitempty"`
 	}
+	NetworkMonitor struct {
```

Comment:
not needed. network monitor runs as separate daemonset..should not be part of cni